deploy/beanstalkdeploy.yaml

```diff
@@ -0,0 +1,25 @@
+#deploy components for Bean stalk deployment. 
+#Using this will require deploycomponents.yaml  
+
+publish:
+  image: briangweber/docker-node:carbon
+  stage: publish
+  script:
+    - grunt bundle --packageVersion=$VERSION --packageName=$CI_PROJECT_NAME
+  only:
+    - master
+  except:
+    - schedules
+
+deploy-int:
+  extends: .deploy
+  stage: deploy-int
+  variables:
+    CI_ENVIRONMENT_NAME: integration
```

Comment:
This is overwriting a built-in GitLab variable. The way to set this one is to have
```yaml
environment:
  name: integration
```

Reply:
https://docs.gitlab.com/ee/ci/environments.html#defining-environments